api/envoy/http/service_control/requirement.proto

```diff
@@ -18,11 +18,11 @@ package google.api.envoy.http.service_control;
 
 import "validate/validate.proto";
 
-// APIKeyLocation defines the location to extract api key.
+// ApiKeyLocation defines the location to extract api key.
```

Comment:
This is a breaking change for API Gateway team. From my understanding, we don't have a way of versioning our protos yet. 

@toddbeckman-g, could you clarify if this is safe to import?

Reply:
Good catch!

Reply:
Oh well maybe just changing the message name is not breaking?

Reply:
We don't really use any of these fields directly through code, so the main question is if the generated json fields remain the same before and after this change.

Reply:
As discussed offline, simple renaming without change field is backward compatible.